bundles/edu.kit.kastel.sdq.eclipse.common.view/src/edu/kit/kastel/eclipse/common/view/ui/TestDetailsDialog.java

```diff
@@ -13,7 +13,7 @@
 import org.eclipse.swt.widgets.Shell;
 import org.eclipse.swt.widgets.Text;
 
-public class TestDetailsDialog extends Dialog {
+class TestDetailsDialog extends Dialog {
 	private static final int WIDTH = 300;
```

Comment:
Does this size actually look good on 4k-screens?

Reply:
It's looks ok as far as I can see. The dialog itself is resizable (or it should). But if we want other sizes as default, we can do this.